BWT/Program.cs

```diff
@@ -0,0 +1,10 @@
+﻿string? input = Console.ReadLine();
+if (string.IsNullOrEmpty(input))
+{
+    Console.WriteLine("EMPTYSTRING");
+}
+
+(string transformedString, int end) = burrowsWheelerTransform.BWTSolver.BurrowsWheelerTransform(input!);
+Console.WriteLine(transformedString + $" {end}");
+string restoredString = burrowsWheelerTransform.BWTSolver.ReverseBurrowsWheelerTransform(transformedString, end);
```

Comment:
Можно проще: лишний раз не указывать типы и при выводе строки оставить только интерполяцию
```suggestion
var (transformedString, end) = burrowsWheelerTransform.BWTSolver.BurrowsWheelerTransform(input!);
Console.WriteLine($"{transformedString} {end}");
var restoredString = burrowsWheelerTransform.BWTSolver.ReverseBurrowsWheelerTransform(transformedString, end);
```